README.md

```diff
@@ -68,9 +76,11 @@ Please keep the following in mind:
 - Please do not add a README or README.md file to the exercise directory. The READMEs are constructed using shared metadata, which lives in the
 [exercism/problem-specifications](https://github.com/exercism/problem-specifications) repository. Further explanation can be found in [fixing-exercise-readmes](https://github.com/exercism/docs/blob/master/language-tracks/exercises/anatomy/readmes.md)
```

Comment:
I think this section is out of date - we should add `README.md` now, if I'm understanding correctly.

See: https://github.com/exercism/meta/issues/15

Reply:
Interesting, I read this as the "exercises" directory shouldn't have a README.

But each exercise should have a README like raindrops.

Reply:
Okay, so it should be suggested to generate a README file [using configlet](https://github.com/exercism/docs/blob/master/language-tracks/exercises/anatomy/readmes.md). There's a issue created to update create-exercise to do this as well.

I'll update the README to make it clear the READMEs should be generated.